<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisConfigDialogClass</class>
 <widget class="QDialog" name="VisConfigDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisConfigDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="VisConfigTable"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
